hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientReplicatedMapProxy.java

```diff
@@ -93,7 +93,7 @@ public ClientReplicatedMapProxy(String serviceName, String objectName) {
     @Override
     protected void onInitialize() {
         int partitionCount = getContext().getPartitionService().getPartitionCount();
-        targetPartitionId = ThreadLocalRandom.current().nextInt(partitionCount);
+        targetPartitionId = ThreadLocalRandomProvider.get().nextInt(partitionCount);
```

Comment:
hmm you've already a PR for that: https://github.com/hazelcast/hazelcast/pull/9690/files